src/Git/CommitFileViaConsole.php

```diff
@@ -26,4 +29,30 @@ public function __invoke(
             $repositoryDirectory
         ))->mustRun();
     }
+
+    /**
+     * @param non-empty-string $repositoryDirectory
+     * @param non-empty-string $filename
+     */
+    private function assertBranch(
```

Comment:
`assertBranch()` naming is a bit generic: would suggest naming this something like `assertWeAreOnBranch...` or similar